packages/task/src/node/task-problem-collector.ts

```diff
@@ -44,4 +44,24 @@ export class ProblemCollector {
         });
         return markers;
     }
+
+    processLineMatchBegin(line: string): boolean {
```

Comment:
maybe we could rename this function? in my opinion what this function does is more like "matchBeginMatcher"

Reply:
fixed, done !